src/VaporServiceProvider.php

```diff
@@ -163,7 +164,7 @@ protected function registerCommands()
             return new VaporWorkCommand($app['queue.vaporWorker']);
         });
 
-        $this->commands(['command.vapor.work']);
+        $this->commands(['command.vapor.work', VaporListFailedCommand::class]);
```

Comment:
maybe we want to share the same style here? Meaning `command.vapor.queue-failed`.